scripts/config.py

```diff
@@ -197,7 +206,11 @@ def custom_parser_options(self):
             Exclude alternative implementations and platform support options, as well as
             some options that are awkward to test.
             """)
-
+        self.add_adapter(
+            'realfull', realfull_adapter,
```

Comment:
I'm not a huge fan of the `realfull` name, because it seems that the other one is a sort of fake one... What about `everything` or `all`? That would be more meaningful to me. 
The question is also open for the 2nd reviewer (@ronald-cron-arm )

Reply:
Yes that's debatable but that's not new, this is what we have currently in mbedtls config.py. We should keep them aligned and changing the name would rather be outside the scope of this PR I'd say.

Reply:
It's not exactly `everything`, because [regardless of why it was created](https://github.com/Mbed-TLS/mbedtls/issues/520#issuecomment-727820890), what it's used for is to generate the documentation. What it does isn't 100% correct, but then our approach can't be 100% correct since enabling `MBEDTLS_xxx_REMOVE_yyy` options causes these options' documentation to be included, but can also cause some functions to be excluded.

We should revive https://github.com/Mbed-TLS/mbedtls/pull/965 …